Main.py

```diff
@@ -28,27 +24,28 @@ def try_get(dictionary, key, lower = False, typer = lambda x: x, default = None)
 #######################################
 default_server_group_id = -1
 
-bot_loop_forever=True
+bot_loop_forever = True
 log.info("Initializing script....")
 while bot_loop_forever:
     try:
         log.info("Connecting to Teamspeak server...")
-        with ThreadsafeTSConnection(Config.user
-                                    , Config.passwd
-                                    , Config.host
-                                    , Config.port
-                                    , Config.keepalive_interval
-                                    , Config.server_id
-                                    , Config.bot_nickname) as ts3conn:
-            BOT=Bot(Config.db_file_name, ts3conn, Config.verified_group, Config.bot_nickname)
-
-            httpserver = ipc.createHTTPServer(BOT, port= Config.ipc_port)
+        with ThreadsafeTSConnection(Config.user,
+                                    Config.passwd,
+                                    Config.host,
+                                    Config.port,
+                                    Config.keepalive_interval,
+                                    Config.server_id,
+                                    Config.bot_nickname
+                                    ) as ts3conn:
+            BOT = Bot(Config.db_file_name, ts3conn, Config.verified_group, Config.bot_nickname)
+
+            httpserver = ipc.createHTTPServer(BOT, port=Config.ipc_port)
             httpserver.start()
 
-            ipcIsPublic = os.getenv("IPC_PUBLIC","false").lower() in ['true', '1', 'y', 'yes']
+            ipcIsPublic = os.getenv("IPC_PUBLIC", "false").lower() in ['true', '1', 'y', 'yes']
             if ipcIsPublic:
                 log.warn("The IPC socket is open to the network, this is only ok in isolated and/or "
-                            "secure environments")
+                         "secure environments")
             """
             IPCS=ipc.TwistedServer(Config.ipc_port, ts3conn,
```

Comment:
The `TwistedServer` should be gone by now. Is this PR based on an older commit?

Reply:
just a leftover. Will clean that up.